build.gradle

```diff
@@ -501,6 +504,7 @@ subprojects {
 
     useJUnitPlatform {
       includeEngines 'junit-jupiter'
+      excludeTags 'quarantined'
```

Comment:
How can we use `deflake.yml` to run flaky tests if they are excluded by the test task? Maybe we should enable `deflake.yml` to also run `quarantinedTest`?

Reply:
Good point, I will update that job.